src/components/editor/property/PanelResourceTopNavItem.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { setCurrentComponent } from 'actions/index'
+import { useDispatch, useSelector } from 'react-redux'
+import { displayResourceValidations } from 'selectors/errors'
+import { selectCurrentComponentKey } from 'selectors/index'
+
+const PanelResourceTopNavItem = (props) => {
+  const dispatch = useDispatch()
+
+  const classNames = ['list-group-item', 'list-group-item-action', 'list-group-top-item']
+  const currentComponentKey = useSelector((state) => selectCurrentComponentKey(state, props.resourceKey))
+  if (props.componentKey === currentComponentKey) classNames.push('active')
+  // Check or space
+  const prefixMark = props.hasValue ? '\u2713' : '\u00A0\u00A0'
+  const displayValidations = useSelector((state) => displayResourceValidations(state))
+  if (displayValidations && props.hasError) classNames.push('text-danger')
+
+  return (<button
```

Comment:
Should completed nested properties propagate up to top level so that the top-level reflects the status of it's  children? 

For example when I load up the **Uber template 1** resource template and  expand the *Uber Template, Property 1* panel, I see the children in the left nav as expected:

<img width="1336" alt="Screen Shot 2020-09-01 at 2 02 20 PM" src="https://user-images.githubusercontent.com/71847/91900352-ca64bc80-ec5b-11ea-9ca6-d976611400fc.png">

When I fill in text for the nested  **Uber Template 2** and **Uber Template 3** the children in the left nav are checked correctly but the top-level **Uber Template 1**  is not checked:

<img width="1313" alt="Screen Shot 2020-09-01 at 1 45 16 PM" src="https://user-images.githubusercontent.com/71847/91900629-33e4cb00-ec5c-11ea-9029-9e56beba4883.png">

I am fine if you want to just ticket this matter to resolve instead of implementing in this PR.

Reply:
I intentionally implemented the simplest possible way (no bubble up), figuring that if users felt this was not optimal then could ticket.

Reply:
Just confirmed with @michelleif that we should expect the status to bubble up. Here is the LD4P/sinopia_editor/issues/2409